mods/ts/rules/ai.yaml

```diff
@@ -20,7 +20,7 @@ Player:
 		PowerTypes: gapowr, napowr, naapwr
 		BarracksTypes: gapile, nahand
 		VehiclesFactoryTypes: gaweap, naweap
-		ProductionTypes: gapile, nahand, gaweap, naweap
+		ProductionTypes: gapile, nahand, gaweap, naweap, gahpad, nahand
```

Comment:
```suggestion
		ProductionTypes: gapile, nahand, gaweap, naweap, gahpad, nahpad
```

Reply:
Huh, thanks. For some reason it still built the Nod helipad and their aircraft so I didn't notice this. :eyes: